src/views/components/MoreSettings.jsx

```diff
@@ -137,6 +141,20 @@ export const MoreSettings = () => {
     }
   };
 
+  const changeName = async () => {
+    try {
+      await hmsActions.updatePeer({
+        name: newName.length > 0 ? newName : undefined,
```

Comment:
if newName.length is not > 0 , we should not make this call

Reply:
Removed the condition altogether as the button is disabled if newName.length < 1